pages.zh/osx/pbpaste.md

```diff
@@ -1,11 +1,12 @@
 # pbpaste
 
-> 将剪贴板的内容发送到标准输出（命令行）。
+> 将剪贴板的内容发送到标准输出。
+> 相当于在键盘上按下 Cmd + V。
```

Comment:
```suggestion
> 相当于在键盘上按下 Cmd + V.
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#chinese-specific-rules (5. Use a half-width punctuation to end a sentence when the last character is half-width.)

Reply:
Thx